src/sentry/replays/blueprints/api.md

```diff
@@ -238,6 +238,47 @@ Deletes a replay instance.
 
 - Response 204
 
+## Replay Accessibility Issues [/projects/<organization_slug>/<project_slug>/replays/<replay_id>/accessibility-issues]
+
+This resource does not accept any URI parameters and is not paginated. Responses are ingested whole.
+
+### Fetch Replay Accessibility Issues [GET]
+
+Retrieve a collection of accessibility issues.
+
+**Attributes**
+
+| Column          | Type                   | Description                                        |
+| --------------- | ---------------------- | -------------------------------------------------- |
+| id              | string                 | -                                                  |
+| timestamp       | number                 | -                                                  |
+| impact          | Optional[enum[string]] | One of: 'minor', 'moderate', 'serious', 'critical' |
+| description     | string                 | -                                                  |
+| help_url        | string                 | -                                                  |
+| element         | string                 | The offending element's DOM selector.              |
+| failure_summary | string                 | -                                                  |
+
+- Response 200
+
+  ```json
+  {
+    "meta": {
+      "total": 1
+    },
+    "data": [
+      {
+        "id": "AHC203525",
```

Comment:
is this an issue id, specific to the detected issue type, mapping to the description/help_url/summary?

Reply:
Unclear to me at the moment.  I just copied what was on the hackweek project.  Its a todo for me to get in there and figure out exactly what axe-core is returning.  the other fields are pretty self-explanatory i feel but `id` is definitely not.

Reply:
np np